packages/workbox-build/src/lib/utils/filter-files.js

```diff
@@ -19,23 +60,28 @@ module.exports = (fileDetails, options) => {
     return true;
   });
 
-  // Convert to manifest format
-  return filteredFileDetails.map((fileDetails) => {
-    let url = fileDetails.file.replace(path.sep, '/');
-
-    // Modify URL Prefix
-    if (options && options.modifyUrlPrefix) {
-      url = modifyUrlPrefixes(url, options.modifyUrlPrefix);
-    }
-
-    if (options.dontCacheBustUrlsMatching &&
-      url.match(options.dontCacheBustUrlsMatching)) {
-      return url;
-    }
-
+  // Take the array of fileDetail objects and convert it into an array of
+  // {url, revision} objects, with path.sep replaced with /.
+  const normalizedManifest = filteredFileDetails.map((fileDetails) => {
     return {
-      url: url,
+      url: fileDetails.file.replace(path.sep, '/'),
       revision: fileDetails.hash,
     };
   });
+
+  const manifestTransforms = options.manifestTransforms || [];
+
+  if (options.modifyUrlPrefix) {
+    manifestTransforms.push(modifyUrlPrefixTranform(options.modifyUrlPrefix));
```

Comment:
I would have run the workbox transforms BEFORE the user transforms. i.e. my thinking of the API was that users would be a final chance to alter the urls / revisions.